dashboard/test/ui/features/step_definitions/account_steps.rb

```diff
@@ -332,3 +332,18 @@ def reset_session
   steps "And I wait for 3 seconds"
   navigate_to replace_hostname('http://studio.code.org/reset_session')
 end
+
+Given /^sign in "([^"]*)" (\d+) times$/ do |name, times|
```

Comment:
Two nits:

1. Please add "I", for parallelism with our other steps; ie, `Given I sign in "Teacher" 10 times` rather than `Given sign in "Teacher" 10 times`
2. Could you also move this block farther up in the file, so it's adjacent to the other "I sign in/out" steps [here](https://github.com/code-dot-org/code-dot-org/blob/687f48de593e21423134aeac2b6c212f354004ad/dashboard/test/ui/features/step_definitions/account_steps.rb#L19-L38)?